public/src/js/constants/defaults.js

```diff
@@ -209,10 +209,10 @@ export default {
     reauthTimeout:         60000,
 
     imageCdnDomain:        '.guim.co.uk',
-    imageCdnDomainExpr:    /^https?:\/\/(.*)\.guim\.co\.uk\//,
-    imgIXDomainExpr:       /^https?:\/\/i\.guim\.co\.uk\/img\/static\//,
+    imageCdnDomainExpr:    /^https:\/\/(.*)\.guim\.co\.uk\//,
+    imgIXDomainExpr:       /^https:\/\/i\.guim\.co\.uk\/img\/static\//,
```

Comment:
There's no need to change this expression because the code rewrite the URL to include https already